src/components/OpportunityCard/OpportunityCard.tsx

```diff
@@ -17,21 +28,140 @@ const bgForType = (type: CardType) => {
+  selected,
+  toggleSelection,
+  name,
+}: {
+  selected: boolean;
+  toggleSelection: () => void;
+  name: string;
+}) => {
+  const { t } = useTranslation();
+
+  const onClick = (e: React.MouseEvent<HTMLElement>) => {
+    e.stopPropagation();
+    e.preventDefault();
+    toggleSelection();
+  };
+
+  const onKeyDown = (e: React.KeyboardEvent) => {
+    if (['Enter', ' '].includes(e.key)) {
+      e.preventDefault();
+      toggleSelection();
+    }
+  };
+
+  return (
+    <span
+      className="inline-flex items-center gap-2 font-arial font-bold text-[12px]/[24px] text-accent"
+      onClick={onClick}
+      onKeyDown={onKeyDown}
+      aria-label={`${t('choose')} ${name}`}
+      role="checkbox"
+      aria-checked={selected}
+      tabIndex={0}
+    >
+      <span aria-hidden>{t('choose')}</span>
+      <span aria-hidden className={`size-5 appearance-none border border-accent ${selected ? 'bg-accent' : ''}`} />
+    </span>
+  );
+};
+
+export const OpportunityCard = ({
+  description,
+  employmentOutlook,
+  hasRestrictions,
+  industryName,
+  matchLabel,
+  matchValue,
+  mostCommonEducationBackground,
+  name,
+  trend,
+  type,
+  selected,
+  toggleSelection,
+}: OpportunityCardProps) => {
+  const { t } = useTranslation();
+  const { sm } = useMediaQueries();
   return (
-    <div className="rounded-[8px] shadow-[0_1px_1px_1px_rgba(0,0,0,0.10)] bg-white pt-5 pr-5 pl-6 pb-7">
-      <div className="flex flex-col sm:flex-row sm:gap-5 gap-3">
-        <Match match={matchValue} label={matchLabel} bg={bgForType(type)} />
-        <div className="flex flex-col">
-          <span className="text-heading-3 text-black">{name}</span>
-          <span className="text-[16px]">{description}</span>
+    <div className="rounded shadow-border bg-white pt-5 pr-5 pl-6 pb-7">
+      <div className="flex flex-col gap-3 sm:flex-row sm:gap-5">
+        <div className="flex flex-row justify-between align-center">
+          <Match match={matchValue} label={matchLabel} bg={bgForType(type)} />
+          {!sm && <SelectedCheckbox selected={selected} toggleSelection={toggleSelection} name={name} />}
+        </div>
+        {!sm && <div className="font-[22px]/[34px] font-bold text-black">{name}</div>}
+        <div className="flex flex-col gap-y-2">
+          {sm && (
+            <div className="flex flex-row justify-between">
+              <span className="grow-1 text-heading-2 text-black">{name}</span>
+              <span className="grow-0 shrink">
+                <SelectedCheckbox selected={selected} toggleSelection={toggleSelection} name={name} />
+              </span>
+            </div>
+          )}
+          <span className="font-arial text-body-md">{description}</span>
+          <div className="mt-4 flex flex-wrap">
+            <BottomBox title={t('tool.competences.trend')}>
+              <span
+                className="material-symbols-outlined text-accent"
+                aria-description={t(`tool.competences.trend-${trend}`)}
```

Comment:
`aria-description` should be `aria-label`.

`aria-description` is not read in this case as the div element is not interactive.